server/index.js

```diff
@@ -0,0 +1,73 @@
+const aws = require('aws-sdk')
+const { parse } = require('json2csv')
+const { Server } = require('socket.io')
+
+const PORT = process.env.PORT || 4000
+
+const userQuestionMatrix = {}
```

Comment:
I don't think tsdx is really designed to run scripts, it's pretty much just for publishing npm packages it seems. I think something like this might work though. https://github.com/TypeStrong/ts-node